DEVELOPMENT.md

```diff
@@ -1,18 +1,65 @@
 # Development
 
-## Localizations
+## Prerequisites for development
 
-### Implementation
+You need to have [Yarn](https://yarnpkg.com/lang/en/) and [Node.js](https://nodejs.org) installed.
+
+## Installing and building
+
+To get started working with this site, clone the repository and install dependencies by running ```yarn``` or ```yarn install```
+
+This project uses Gatsby, so to start a development server, run:
+
+```
```

Comment:
adding ```bash will highlight the code in some cases, not sure if it would change anything here.

Reply:
Doesn't seem to affect the output in this case.